### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by joy-yao**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.124.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ General Changes
 ---------------
 
 * The :func:`approx_percentile` aggregation now also accepts an array of percentages.
+* Allow nested rowtype reference
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.124.rst]: Release notes should end in period

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.124.rst]: reference -> references

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.124.rst]: "Allow nested row type references."

**Comment by joy-yao** [comment]
Thanks @electrum . Updated

**Pull request merged**